src/Cli/dotnet/commands/dotnet-workload/update/LocalizableStrings.resx

```diff
@@ -160,6 +160,6 @@
     <value>No workload update found.</value>
   </data>
   <data name="WorkloadVersionRequestedNotFound" xml:space="preserve">
-    <value>Workload version {0} not found.</value>
+    <value>Workload version {0} not found. Adding a feed that contains it to your NuGet.config may help.</value>
```

Comment:
The message now uses `WorkloadVersionFromGlobalJsonNotFound`, and the text is:

> Workload update failed: Workload version 8.0.300-preview.0.24217.2, which was specified in C:\SdkTesting\global.json, was not found. Run "dotnet workload restore" to install this workload version.

I don't think we need a message here about adding a feed, as in most cases the workload should be available on NuGet.org and they shouldn't need to mess with their feeds.  If the `dotnet workload restore` command fails, that's when we could mention the feeds.  It looks like that message could stand to be cleaned up, currently it looks something like this:

> Unhandled exception: System.IO.FileNotFoundException: Workload version 8.0.300-preview.0.24217.2, which was specified in C:\SdkTesting\global.json, was not found. Run "dotnet workload restore" to install this workload version.
   at Microsoft.NET.Sdk.WorkloadManifestReader.SdkDirectoryWorkloadManifestProvider.ThrowExceptionIfManifestsNotAvailable() in C:\git\dotnet-sdk\src\Resolvers\Microsoft.NET.Sdk.WorkloadManifestReader\SdkDirectoryWorkloadManifestProvider.cs:line 231
   at Microsoft.NET.Sdk.WorkloadManifestReader.SdkDirectoryWorkloadManifestProvider.GetManifests() in C:\git\dotnet-sdk\src\Resolvers\Microsoft.NET.Sdk.WorkloadManifestReader\SdkDirectoryWorkloadManifestProvider.cs:line 269
   at Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver.LoadManifestsFromProvider(IWorkloadManifestProvider manifestProvider) in C:\git\dotnet-sdk\src\Resolvers\Microsoft.NET.Sdk.WorkloadManifestReader\WorkloadResolver.cs:line 124
   at Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver.InitializeManifests() in C:\git\dotnet-sdk\src\Resolvers\Microsoft.NET.Sdk.WorkloadManifestReader\WorkloadResolver.cs:line 91
   at Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver.GetInstalledManifests() in C:\git\dotnet-sdk\src\Resolvers\Microsoft.NET.Sdk.WorkloadManifestReader\WorkloadResolver.cs:line 771
   at Microsoft.DotNet.Workloads.Workload.Update.WorkloadUpdateCommand.Execute() in C:\git\dotnet-sdk\src\Cli\dotnet\commands\dotnet-workload\update\WorkloadUpdateCommand.cs:line 86
   at Microsoft.DotNet.Cli.WorkloadUpdateCommandParser.<>c.<ConstructCommand>b__6_0(ParseResult parseResult) in C:\git\dotnet-sdk\src\Cli\dotnet\commands\dotnet-workload\update\WorkloadUpdateCommandParser.cs:line 52
   at System.CommandLine.Invocation.AnonymousCliAction.Invoke(ParseResult parseResult)
   at System.CommandLine.Invocation.InvocationPipeline.Invoke(ParseResult parseResult)
   at System.CommandLine.ParseResult.Invoke()
   at Microsoft.DotNet.Cli.Program.ProcessArgs(String[] args, TimeSpan startupTime, ITelemetry telemetryClient) in C:\git\dotnet-sdk\src\Cli\dotnet\Program.cs:line 232